newFile.txt

```diff
@@ -0,0 +1 @@
+This is just for an example on how pull requests work.
```

Comment:
I think that we should add an exclamation point instead of a period.

Reply:
I disagree but okay